### ISSUE TITLE
Javadoc can't be generated [Java]

### ISSUE THREAD

**Issue by damiencarol** [dmlc/xgboost#676]
Javadoc can't be genrated for the Java wrapper.
```
[dcarol@dcarol xgboost4j]$ mvn package javadoc:javadoc -DskipTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building xgboost4j 1.1
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ xgboost4j ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ xgboost4j ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ xgboost4j ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/dcarol/work/git/xgboost/java/xgboost4j/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ xgboost4j ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ xgboost4j ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ xgboost4j ---
[INFO] 
[INFO] >>> maven-javadoc-plugin:2.10.3:javadoc (default-cli) > generate-sources @ xgboost4j >>>
[INFO] 
[INFO] <<< maven-javadoc-plugin:2.10.3:javadoc (default-cli) < generate-sources @ xgboost4j <<<
[INFO] 
[INFO] --- maven-javadoc-plugin:2.10.3:javadoc (default-cli) @ xgboost4j ---
[INFO] 
Loading source files for package org.dmlc.xgboost4j...
Loading source files for package org.dmlc.xgboost4j.wrapper...
Loading source files for package org.dmlc.xgboost4j.util...
Constructing Javadoc information...
Standard Doclet version 1.8.0_66
Building tree for all the packages and classes...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/Booster.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/DMatrix.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/DMatrix.SparseType.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/IEvaluation.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/IObjective.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/XgboostJNI.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/CVPack.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/ErrorHandle.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/Initializer.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/NativeUtils.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/Trainer.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/XGBoostError.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/overview-frame.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/package-frame.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/package-summary.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/package-tree.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/package-frame.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/package-summary.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/package-tree.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/package-frame.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/package-summary.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/package-tree.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/constant-values.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/serialized-form.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/class-use/IObjective.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/class-use/IEvaluation.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/class-use/Booster.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/class-use/DMatrix.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/class-use/DMatrix.SparseType.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/class-use/XgboostJNI.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/Trainer.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/XGBoostError.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/ErrorHandle.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/Initializer.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/NativeUtils.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/class-use/CVPack.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/package-use.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/util/package-use.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/org/dmlc/xgboost4j/wrapper/package-use.html...
Building index for all the packages and classes...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/overview-tree.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/index-all.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/deprecated-list.html...
Building index for all classes...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/allclasses-frame.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/allclasses-noframe.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/index.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/overview-summary.html...
Generating /home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs/help-doc.html...
2 errors
84 warnings
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.231 s
[INFO] Finished at: 2015-12-07T13:31:13+01:00
[INFO] Final Memory: 16M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.3:javadoc (default-cli) on project xgboost4j: An error has occurred in JavaDocs report generation:
[ERROR] Exit code: 1 - /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:106: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:115: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:130: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:141: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:154: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:169: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:180: warning: no description for @param
[ERROR] * @param evalMatrixs
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:181: warning: no description for @param
[ERROR] * @param evalNames
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:182: warning: no description for @param
[ERROR] * @param iter
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:183: warning: no description for @param
[ERROR] * @param eval
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:185: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:205: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:216: warning: no description for @param
[ERROR] * @param evalMat
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:217: warning: no description for @param
[ERROR] * @param evalName
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:218: warning: no description for @param
[ERROR] * @param iter
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:220: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:262: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:273: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:285: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:300: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:308: warning: no description for @param
[ERROR] * @param modelPath
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:322: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:339: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:356: warning: no description for @throws
[ERROR] * @throws FileNotFoundException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:357: warning: no description for @throws
[ERROR] * @throws UnsupportedEncodingException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:358: warning: no description for @throws
[ERROR] * @throws IOException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:359: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:383: warning: no description for @throws
[ERROR] * @throws FileNotFoundException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:384: warning: no description for @throws
[ERROR] * @throws UnsupportedEncodingException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:385: warning: no description for @throws
[ERROR] * @throws IOException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:386: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:407: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:436: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:62: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/Booster.java:76: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:113: warning: no description for @param
[ERROR] * @param labels
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:115: warning: no @throws for org.dmlc.xgboost4j.util.XGBoostError
[ERROR] public void setLabel(float[] labels) throws XGBoostError {
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:121: warning: no description for @param
[ERROR] * @param weights
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:122: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:131: warning: no description for @param
[ERROR] * @param baseMargin
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:132: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:141: warning: no description for @param
[ERROR] * @param baseMargin
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:142: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:151: warning: no description for @param
[ERROR] * @param group
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:152: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:173: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:182: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:191: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:199: warning: no description for @param
[ERROR] * @param rowIndex
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:201: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:214: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:224: warning: no description for @param
[ERROR] * @param filePath
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:93: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:72: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:54: warning: no description for @param
[ERROR] * @param dataPath
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/DMatrix.java:55: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/IEvaluation.java:31: warning: no description for @param
[ERROR] * @param predicts
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/IEvaluation.java:32: warning: no description for @param
[ERROR] * @param dmat
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/IEvaluation.java:33: warning: no description for @return
[ERROR] * @return
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:53: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:63: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:72: warning: no description for @return
[ERROR] * @return
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:73: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:83: warning: no description for @return
[ERROR] * @return
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:84: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/CVPack.java:40: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/ErrorHandle.java:43: warning: no description for @throws
[ERROR] * @throws org.dmlc.xgboost4j.util.XGBoostError
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/Initializer.java:67: warning: no description for @param
[ERROR] * @param libPath
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/Initializer.java:68: warning: no description for @throws
[ERROR] * @throws IOException
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:50: error: no tag name after @
[ERROR] * @throws IllegalArgumentException If the path is not absolute or if the filename is shorter than three characters (restriction of {@see File#createTempFile(java.lang.String, java.lang.String)}).
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:29: error: unexpected text
[ERROR] * @see http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "58" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "47" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "47" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "47" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "47" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/NativeUtils.java:33: warning - Tag @see:illegal character: "45" in "http://adamheinrich.com/2012/how-to-load-native-jni-library-from-jar"
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/Trainer.java:49: warning: no @throws for org.dmlc.xgboost4j.util.XGBoostError
[ERROR] public static Booster train(Iterable<Entry<String, Object>> params, DMatrix dtrain, int round,
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/Trainer.java:115: warning: no @throws for org.dmlc.xgboost4j.util.XGBoostError
[ERROR] public static String[] crossValiation(Iterable<Entry<String, Object>> params, DMatrix data, int round, int nfold, String[] metrics, IObjective obj, IEvaluation eval) throws XGBoostError {
[ERROR] ^
[ERROR] /home/dcarol/work/git/xgboost/java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/Trainer.java:152: warning: no @throws for org.dmlc.xgboost4j.util.XGBoostError
[ERROR] public static CVPack[] makeNFold(DMatrix data, int nfold, Iterable<Entry<String, Object>> params, String[] evalMetrics) throws XGBoostError {
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /usr/lib/jvm/java-8-openjdk/jre/../bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/dcarol/work/git/xgboost/java/xgboost4j/target/site/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

**Comment by terrytangyuan** [comment]
Can you try the most recent version from Github?

**Comment by damiencarol** [comment]
I see no change on these files. Same error on same files.
I will make a PR if I have time.

**Comment by yanqingmen** [comment]
hi, @damiencarol; Maybe this error was caused by the version setting of maven javadoc plugin; it works well on my machine. It would be great if you can make a PR to fix this.

**Comment by damiencarol** [comment]
@yanqingmen Yes sir ! :) => #733

**Comment by terrytangyuan** [comment]
Closed via #733
